solr/core/src/java/org/apache/solr/servlet/HttpSolrCall.java

```diff
@@ -893,7 +893,7 @@ private void handleAdminRequest() throws IOException {
             handler != null
                 ? MarkerFactory.getMarker(handler.getClass().getName())
                 : MarkerFactory.getMarker(HttpSolrCall.class.getName()),
-            solrResp.getToLogAsString("[admin]"));
+            solrResp.getToLogAsString("admin"));
```

Comment:
Instead, could `addToLog("admin", true)` be called at some early point to ensure it shows up (first)?

Reply:
I see there is another place where #getToLogAsString is called with a different prefix: `rspObject.getToLogAsString("TaskId: " + this.taskId);`